connexion/json_schema.py

```diff
@@ -62,12 +65,22 @@ def __call__(self, uri):
             return yaml.load(fh, ExtendedSafeLoader)
 
 
-handlers = {
-    "http": URLHandler(),
-    "https": URLHandler(),
-    "file": FileHandler(),
-    "": FileHandler(),
-}
+def resource_from_spec(spec: t.Dict[str, t.Any]) -> Resource:
+    return Resource.from_contents(spec, default_specification=DRAFT4)
+
+
+def retrieve(uri: str) -> Resource:
+    parsed = urllib.parse.urlsplit(uri)
```

Comment:
Would you please consider adding pydoc for this new function?

Reply:
Done.